src/main/java/net/dancier/chatdancer/ChatService.java

```diff
@@ -0,0 +1,17 @@
+package net.dancier.chatdancer;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class ChatService {
+
+ private final ChatDao chatDao;
+ private final ChatDaoInMemory chatDaoInMemory;
+
+ public Chat createNewChat(ChatCreatedResponseDto chatCreatedResponseDto) {
```

Comment:
I would need some clarification on that, not here necessarily, but maybe in person. Why should we not use them in this layer and how exactly does it helps us when we map them? What are the differences?

Reply:
Short answer:
This would tie the presentation layer to the business layer. And we in general want to avoid this. This business layer should be stable on the way our API is designed.
You can also arrange a slot maybe today, to talk about that ;-)